### DISCUSSION TITLE
Table of contents  - top-level README.md only?

### DISCUSSION THREAD

**Discussion by papadeltasierra** [community/community#64042] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Have I misunderstood, or am I misusing, GitHub's table of content support?  It seems to work find for my "top-level" `README.md` but I have files in a `docs/Useful.md` subdirectory and although **Preview** shows the markdown nicely formatted, there is no dropdown table of contact that I could spot.

**Comment by 0xChintan** [comment]
You're not misunderstanding GitHub's table of contents (TOC) support, but it typically generates dropdown-style TOCs for the top-level README.md file. Subdirectory files may not have interactive TOCs by default, so you might need to create TOCs manually for those files.
and  If you want interactive dropdown TOCs in subdirectory files, you might need to consider third-party tools or extensions. For instance, you can use VSCode extensions like "Markdown All in One" or "Markdown TOC" to generate TOCs and then push the Markdown file to GitHub.
